web/html/html-structure/html-structure-define.md

```diff
@@ -28,52 +28,55 @@ The typical structure of HTML elements are defined as:
 
-
 ---
 ## Practice
 
 Identify the terms defining this HTML code:
 
-https://img.enkipro.com/26a6a342cc716551cb0fa5950d2da503.png
-
-A: ???
-B: ???
-C: ???
-D: ???
-E: ???
-F: ???
+```html
+<a href="https:/enki.com/">Enki</a>        
 
-* element
-* opening tag
-* end tag
-* attribute
-* attribute's name
-* attribute's value
-* content
-* value
-* rule
+element: ???
+opening tag: ???
+end tag: ???
+attribute: ???
+attribute's name: ???
+attributes value: ???
+```
 
+* `<a href="https:/enki.com/">Enki</a>`
+* `<a href="https:/enki.com/">`
+* `</a>`
+* `href="https:/enki.com/"`
+* `href`
+* `"https:/enki.com/"`
 
 ---
 ## Revision
 
 Match the terms defining this HTML code:
 
-https://img.enkipro.com/26a6a342cc716551cb0fa5950d2da503.png
+```html
+<a href="https:/enki.com/">
+    Enki
+</a>
+
 
-A: ???
-B: ???
-C: ???
-D: ???
-E: ???
-F: ???
+<a href="https:/enki.com/">Enki</a> : ???
+<a href="https:/enki.com/">: ???
+</a>: ???
+href="https:/enki.com/": ???
+href: ???
+"https:/enki.com/": ???
+Enki: ???
+```
 
 * element
 * opening tag
 * end tag
 * attribute
-* attribute's name
-* attribute's value
+* attributes name
```

Comment:
Wasn't the original text correct? It's "name of an attribute" or "attribute's name", no?